docs/changelog/92118.yaml

```diff
@@ -0,0 +1,16 @@
+pr: 92118
+summary: Allow more than one KNN search clause
+area: Vector Search
+type: enhancement
+issues:
+ - 91187
+highlight:
+  title: Allow more than one KNN search clause
+  body: "It makes sense to allow more than one KNN search clause per individual\n\
+    search request. It may be that different documents have separate vector\nspaces\
+    \ or that a single doc is index with more than one vector space. In\nboth of these\
+    \ scenarios, users may want to retrieve a resulting set that\ntakes into account\
+    \ all their indexed vector spaces. \n\nA prime example here would be searching\
+    \ a semantic text embedding along\nwith searching an image embedding. \n\ncloses\
+    \ https://github.com/elastic/elasticsearch/issues/91187"
```

Comment:
@giladgal would you like to review the text associated with making this change a release highlight?